pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -82,7 +82,7 @@ let
     pname = "paperless-ngx-frontend";
     inherit version src;
 
-    npmDepsHash = "sha256-wUlybMxnXLNmeu2z+RFFOHVEhH12XD3ZfMo5K+HSBpY=";
+    npmDepsHash = "sha256-XTk4DpQAU/rI2XoUvLm0KVjuXFWdz2wb2EAg8EBVEdU=";
 
     nativeBuildInputs = [
```

Comment:
GitHub won't let me post a suggestion for those lines, because this PR didn't touch them...

Good ol' patch:
```patch
diff --git a/pkgs/applications/office/paperless-ngx/default.nix b/pkgs/applications/office/paperless-ngx/default.nix
index a60f926278e..c73bf3c6106 100644
--- a/pkgs/applications/office/paperless-ngx/default.nix
+++ b/pkgs/applications/office/paperless-ngx/default.nix
@@ -14,6 +14,8 @@
 , unpaper
 , poppler_utils
 , liberation_ttf
+, stdenv
+, xcbuild
 }:
 
 let
@@ -86,7 +88,7 @@ let
 
     nativeBuildInputs = [
       python3
-    ];
+    ] ++ lib.optional stdenv.isDarwin xcbuild.xcrun;
 
     postPatch = ''
       cd src-ui
```

I have no way to test this, but it might fix the frontend build on `darwin` (see ci checks/ofborg)

Reply:
I tried this on my machine a few months ago and it did not work. Probably needs more insight.

Reply:
Thanks for the suggestion @IndeedNotJames, but besides the fact that I don't have access to a Mac of any sort (especially none that has nix on it) and that this is a different topic I think that's out of scope for an update PR, especially since darwin builds were (to my knoweledge) already broken prior to this update.
That's why I think a separate PR would be the right place for this.

Reply:
I wasn't aware there is an ongoing build issue on `darwin`.
But if this is an ongoing issue, why isn't this package marked as broken on `darwin` via `meta.broken = stdenv.isDarwin`? :eyes:

Reply:
My bad for dismissing this as "always has been broken". I've went back and took a look at various old updates. It seems as if the tests on darwin have been mostly failing ever since paperless-ngx [came to nixpkgs](https://github.com/NixOS/nixpkgs/pull/168288/checks?check_run_id=5994130492), but that usually appears to be due to [tests](https://github.com/NixOS/nixpkgs/pull/192260/checks?check_run_id=8569525352) [timing out](https://github.com/NixOS/nixpkgs/pull/209591/checks?check_run_id=10504338496).

But yeah, after looking at the logs again, you'ree right that this is fresh breakage and not the "always been broken" I assumed it to be. But I am unable to test / implement your proposed changes (lack of a darwin device), so I sadly can't clean up behind myself in this case :/

Again, apologies for being dismissive.